### ISSUE TITLE
Internal Team Communication

### ISSUE THREAD

**Issue by rickbau5** [uidaho/squireproject#31]
TL:DR - We need to communicate more, and find a platform that we can do that on! I suggest Slack, see below :+1: 

I think the team's communication could be dramatically improved. As it stands, the only correspondence we do outside of our meetings is either through C9 or now increasingly GitHub. C9 was a great solution while most of our work was done there, but now I find myself on there far less than before. 

However, GitHub is not designed for quick and efficient communication. Yes you can @mention people, but all you get is an email, and for me I don't even see them in my main mail box as I get too many GitHub notifications a day to not bundle them quietly. 

As we get deeper into this, I would really prefer if there was a platform that we all actively used to communicate with each other. Several times I found myself today wishing I could get in immediate contact with @brec9824 or @backburn.

With these things in mind, I propose we seek a new platform for communication. We do have a Skype room as per the Contact document on C9, but there doesn't seem to be much activity in there. I think we should push using that, or if there is another platform that would work better for everyone we should definitely discuss it. 

**If you don't read anything else, start here!**
I personally use [Slack](http://www.slack.com) and really like it. I've also used [HipChat](https://www.hipchat.com) (at a company during an internship) but really prefer Slack over it. Some of the benefits I see to having Slack:
- **Mobile app:** the app is great (iOS and Android) and would allow us to see messages immediately - if you @mention someone, they get a notification from the app. Furthermore, it can be run through your web browser too, so you don't _have_ to install the program if you don't want to.
- **Rooms**: Slack is organized into rooms, which we may not use to its fullest, however I envision us having a General Discussion room, a room for [GitHub](https://slack.com/apps/A0F7YS2SX-github) & Travis integration, and maybe a Help Me room.
- **Integrations**: Slack has a bunch of "integrations" as they call it. Basically allow you to customize Slack with apps. For instance, we could hook it up to this repo, and every time someone makes a pull request, commit, comments, etc, it would post a message with a link in that room.
- **It's free**! I think. Or cheap. 
- Many companies use programs like Slack or HipChat for internal communications, why not familiarize yourselves with it now!

Here's a picture of Slack:
![image](https://cloud.githubusercontent.com/assets/11299264/14375397/dcf2bcec-fd15-11e5-9c05-8a2613891828.png)

This is an open discussion! I won't force this Slack thing unless there is interest for it :+1: 

Mention everyone so it actually get's your attention. Maybe. @backburn @drspaceman0 @mora5651 @brec9824 @bolt1003

**Comment by brec9824** [comment]
+1 I definitely think we need to get our team communicating more because it has been lacking. I don't really have a preference for any comm software so it doesn't matter to me what we use. Slack sounds good to me.

**Comment by rickbau5** [comment]
Cool cool. Let's get some more opinions on this. I really think Slack is a great option :+1: 

Also just a heads up, I won't be at the meeting this afternoon. I'll probably be online by then, but i won't be in moscow. I plan on working on the Project Ideas stuff more. I've made a temp project page, and will be adding the create project flow, and modifying the projectfinder page to retrieve actual projects.

**Comment by drop-table-lol** [comment]
Looks good to me, I don't have much of a preference either, but I think you're both right in pushing for more communication.

**Comment by rickbau5** [comment]
Ok so it is free, I went ahead and reserved the domain. You can sign up here:  https://uidaho.slack.com/x-33299441681-33292950496/signup.

**Comment by rickbau5** [comment]
@uidaho/lambda 
Ok so it looks like we're adopting this! Please start using it, especially on a day like today when we're all working on stuff :) Just have it open so we can message each other! I'll be handing out free :peach:s to everyone who is active on [our Slack](http://uidaho.slack.com)!

**Comment by rickbau5** [comment]
Well it aint gunna get any better at this point, closing.
